spec/form_models/job_specification_form_spec.rb

```diff
@@ -0,0 +1,54 @@
+require 'rails_helper'
+RSpec.describe JobSpecificationForm, type: :model do
+  subject { JobSpecificationForm.new({}) }
+
+  context 'validations' do
+    it { should validate_presence_of(:job_title) }
+    it { should validate_presence_of(:headline) }
+    it { should validate_presence_of(:job_description) }
+    it { should validate_presence_of(:minimum_salary) }
+    it { should validate_presence_of(:working_pattern) }
+
+    describe '#minimum_salary' do
+      describe '#minimum_salary_lower_than_maximum' do
+        let(:job_specification) do
+          JobSpecificationForm.new(headline: 'headline', job_title: 'job title',
+                                   job_description: 'description', working_pattern: :full_time,
+                                   minimum_salary: 20, maximum_salary: 10)
+        end
+
+        it 'the minimum salary should be less than the maximum salary' do
+          expect(job_specification.valid?).to be false
+          expect(job_specification.errors.messages[:minimum_salary][0])
+            .to eq('must be lower than the maximum salary')
+        end
+      end
+    end
+  end
+
+  context 'when all attributes are valid' do
```

Comment:
@tahb  I don't think I do have a failing scenario in `ApplicationDetailsForm`  are we looking in different places?

Reply:
I cannot find where I was my bad for not linking it. A failing test seems like a good idea for them mind?

Reply:
Since all the validation failures are individually tested I'm not sure of the value of another scenario catering for that. I added this to make sure that all the fields were populated correctly. What do you recommend?